x/blob/types/params.pb.go

```diff
@@ -68,7 +68,7 @@ func (m *Params) GetGasPerBlobByte() uint32 {
 	return 0
 }
 
-func (m *Params) GetGovSquareSizeUpperBound() uint64 {
+func (m *Params) GetGovMaxSquareSize() uint64 {
```

Comment:
This one should have been caught by CI... weird

Reply:
It looks like https://github.com/celestiaorg/celestia-app/pull/1768 didn't modify a `.proto` file so the protobuf workflow wasn't run.

Ref: https://github.com/rootulp/celestia-app/blob/0464cdb652b165c25d4f963f60fe8848c2cfeb30/.github/workflows/protobuf.yml#L2

Reply:
Then I propose to run this workflow if a `.proto` changes and also if any `*.pb.*` so that if any change happens on any of these, we check if the proto files are consistent with the generated ones

Reply:
Yea that works! Since proto-lint and proto-gen take much less time than tests, I proposed always running them in https://github.com/celestiaorg/celestia-app/pull/1950